test/usage.js

```diff
@@ -877,9 +877,9 @@ describe('usage tests', function () {
     it('should allow a function to be provided, rather than a number', function () {
```

Comment:
I think it might be worth adding one more test, for:

```js
.version('version', '1.0.0');
```

`accepts version option as first argument, and version number as second argument`.

We'll eventually add additional tests for:

```js
.version()

// and

.version('version')
```